cli/src/cmd/forge/fourbyte.rs

```diff
@@ -0,0 +1,119 @@
+use crate::{
+    cmd::forge::build::{CoreBuildArgs, ProjectPathsArgs},
+    compile,
+    opts::forge::CompilerArgs,
+};
+use clap::Parser;
+use ethers::prelude::artifacts::{output_selection::ContractOutputSelection, LosslessAbi};
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use tracing::trace;
+
+#[derive(Serialize, Debug)]
+struct ImportRequest {
+    #[serde(rename = "type")]
+    import_type: String,
+    data: Vec<LosslessAbi>,
+}
+
+#[derive(Deserialize, Debug)]
+struct ImportTypeData {
+    imported: HashMap<String, String>,
+    duplicated: HashMap<String, String>,
+}
+
+#[derive(Deserialize, Debug)]
+struct ImportData {
+    function: ImportTypeData,
+    event: ImportTypeData,
+}
+
+#[derive(Deserialize, Debug)]
+struct ImportResponse {
+    result: ImportData,
+}
```

Comment:
can we move this under the `UploadSelectorsArgs` struct and impl please?

I like the cmd modules to start with the actual command

Reply:
yep done